tests/js/spec/views/settings/organizationIntegrations/index.spec.jsx

```diff
@@ -83,7 +83,7 @@ describe('OrganizationIntegrations', () => {
     });
 
     publishedSentryAppsRequest = Client.addMockResponse({
-      url: '/sentry-apps/',
+      url: `/sentry-apps/?status=${'published'}`,
```

Comment:
You don't need to interpolate 'published' here. You can just do `?status=published`.